job_definitions/index_services.yml

```diff
@@ -21,13 +28,13 @@
             condition: UNSTABLE_OR_WORSE
             predefined-parameters: |
               USERNAME=index-services
-              JOB=Index services {{ environment }}
+              JOB=Index services {{ environment }} :jenkins_parrot:
```

Comment:
We don't usually put emoji's in the job name (the icon is the thing we usually change). Was that the intent?